tests/e2e/parameters/infrastructure.py

```diff
@@ -0,0 +1,103 @@
+from pyclbr import Function
+
+from aws_cdk import CfnOutput
+from aws_cdk import aws_appconfig as appconfig
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_ssm as ssm
+
+from tests.e2e.utils.infrastructure import BaseInfrastructure
+
+
+class ParametersStack(BaseInfrastructure):
+    def create_resources(self):
+        functions = self.create_lambda_functions()
+        self._create_parameter_string(function=functions["ParameterStringHandler"])
+        self._create_app_config(function=functions["ParameterAppconfigFreeformHandler"])
+
+    def _create_parameter_string(self, function: Function):
+        parameter = ssm.StringParameter(
+            self.stack, id="string_parameter", parameter_name="sample_string", string_value="Lambda Powertools"
+        )
+
+        parameter.grant_read(function)
+
+        CfnOutput(self.stack, "ParameterString", value=parameter.parameter_name)
+        CfnOutput(self.stack, "ParameterStringValue", value=parameter.string_value)
+
+    def _create_app_config(self, function: Function):
+
+        cfn_application = appconfig.CfnApplication(self.stack, id="appconfig-app", name="appe2e", description="appe2e")
```

Comment:
Same as my other comment - do you think we can run into an issue with the name being hard coded?

e.g., parallel tests, tests that failed due to networking and left orphan resources, etc.

Reply:
@heitorlessa thanks for bringing all these insights! This is very important so that I don't make these mistakes again in the future. :heart: